.travis.yml

```diff
@@ -12,7 +12,7 @@ script:
 - npm test
 - npm run build
 env:
-  - SITE_BASEURL_PRODUCTION="https://labs.usa.gov/" SITE_BASEURL_STAGING="https://ffd-microsite-staging.apps.cloud.gov/"
+  - SITE_BASEURL_PRODUCTION="https://ffd-microsite-production.apps.cloud.gov/" SITE_BASEURL_STAGING="https://ffd-microsite-staging.apps.cloud.gov/"
```

Comment:
Tried separating to different lines but it seems like travis parsed things differently and one of the build was failing.